ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/NetworkProtocolVersion.hs

```diff
@@ -25,9 +25,12 @@ instance HasNetworkProtocolVersion (ShelleyBlock era) where
 
 instance SupportedNetworkProtocolVersion (ShelleyBlock era) where
   supportedNodeToNodeVersions   _ = Map.fromList [
-        (NodeToNodeV_1, ShelleyNodeToNodeVersion1)
+        (NodeToNodeV_1, ShelleyNodeToNodeVersion1),
         -- V_2 enables block size hints for Byron headers and the hard fork
         -- combinator, unused by Shelley-only
+        (NodeToNodeV_2, ShelleyNodeToNodeVersion1),
+        (NodeToNodeV_3, ShelleyNodeToNodeVersion1),
+        (NodeToNodeV_4, ShelleyNodeToNodeVersion1)
       ]
```

Comment:
I don't think we still care about compatibility with Shelley-only testnets, so we could even just have `(NodeToNodeV_4, ShelleyNodeToNodeVersion1)` as the single supported version.

Reply:
Done